### DISCUSSION TITLE
interpolated WPF

### DISCUSSION THREAD

**Discussion by Ludwa86** [dotnet/ResXResourceManager#672] [Q&A]
Hello,
I would like to ask for help.
I have it defined in ResXResourceManager

![Image](https://github.com/user-attachments/assets/ae2eb805-a14b-466e-8c0a-72918f4a9ccd)

and I would like to display text with an index in WPF.

`<Label Grid.Row="0" Grid.Column="0" Content="{Binding Path=txtImage, Source={StaticResource Resources}}"/>`

The label should display text with an index of 1-3.
I couldn't find a working entry.

Thank you

**Comment by tom-englert** [comment]
`string.Format` in your view-model?

**Reply by Ludwa86** [reply]
Is it possible to be more specific?
I don't understand what you mean. The text is written from the resx library.
The text is static, it changes according to the selected language. It is displayed only in XML, I do not change the text in the C# code.

Thank you
